### PR TITLE
change default driver to wd

### PR DESCRIPTION
**Pull request by aotarola**
**BREAKING CHANGE**: This change will force existent clients
to update their code to either explicitly use the sync
driver or use the async API

---
_This PR was started by: [git wf pr](https://github.com/groupon/git-workflow/releases/tag/v1.0.3)_

### CODE REVIEW

File: test/testium-core.test.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 import { getTestium, getBrowser } from '../';
 
 const gofer = new Gofer({
   globalDefaults: {},
 });
-function fetch(uri, options) {
-  return Bluebird.resolve(gofer.fetch(uri, options));
+
+async function fetch(uri, options) {
+  return gofer.fetch(uri, options);
```
Comment (markowsiak) [inline test/testium-core.test.js]: Do we need `await` statements here?

File: test/testium-core.test.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 }
-function fetchResponse(uri, options) {
-  return Bluebird.resolve(gofer.fetch(uri, options).getResponse());
+async function fetchResponse(uri, options) {
+  return gofer.fetch(uri, options).getResponse();
```
Comment (markowsiak) [inline test/testium-core.test.js]: similarly here

File: test/testium-core.test.js (hunk 1 above)
Comment (dbushong) [reply test/testium-core.test.js]: no, we just don't need the `async` :-)

Comment (dbushong) [reply test/testium-core.test.js]: equivalent:
```js
const fetch = gofer.fetch.bind(gofer);
```

**Comment by dbushong** [comment]
could you throw a `chore: npm audit fix` into this pr?

**Pull request merged**